src/config/logger.ts

```diff
@@ -91,23 +82,32 @@ export const overrideProbotLoggingMethods = (probotLogger: Logger) => {
 };
 
 const createNewLogger = (name: string, options: Partial<LoggerOptions> = {}): Logger => {
-	return createLogger(
-		{
-			name,
-			stream: LOG_STREAM,
-			level: globalLoggingLevel,
-			serializers: {
-				err: errorSerializer,
-				res: responseSerializer,
-				req: requestSerializer
-			},
-			...options
-		});
+	return createLogger({
+		name,
+		stream: LOG_STREAM,
+		level: globalLoggingLevel,
+		serializers: {
+			err: errorSerializer,
+			error: errorSerializer,
+			res: responseSerializer,
```

Comment:
I assume this is the place that fix the `circular` json stuff, as mostly `res`, `req` has circular reference inside and this `serializer` will use that `safe-json-stringify`

-----

But just to double check, this fix wont' fix the error if someone log a circular json with some other names right?, like bellow, or am I missing something?

```
const data = {
				a: { foo: true },
				b: { bar: "string" }
			};
data.a = data as any;
req.log('whatever', { data } );
```

Reply:
yes, it will.  bunyan uses it as a fallback if it's own circular dependency check fails to catch it (which I've seen in a few occasions).  I have a test that does exactly this.